### ISSUE TITLE
How to retrieve authorization code in a Reactjs application

### ISSUE THREAD

**Issue by Nessario** [AzureAD/microsoft-authentication-library-for-js#2474]
## Library
- [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
- [x] `@azure/msal-browser@2.x.x`
- [ ] `@azure/msal-angular@0.x.x`
- [ ] `@azure/msal-angular@1.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

## Description
Hello, so we were looking into using the msal-browser to retrieve the authorization code of the user, from which then we have access to update the user's calendar events. But, for some reason the client always is failing to return data only returning `TypeError: errorCode is null` and not much more than that.

Below, I'll attach the configuration that we are using to call those client methods:
```
import { PublicClientApplication } from "@azure/msal-browser";
...
const client = new PublicClientApplication({
    auth: {
        authority: "https://login.microsoftonline.com/organizations/",
        clientId: libConfig.appId,
        redirectUri: libConfig.redirectUri,
    },
    cache: {
        cacheLocation: "sessionStorage",
        storeAuthStateInCookie: true,
    },
});

const code = await client.acquireTokenPopup({
    scopes: [
        "user.read",
        "profile",
        "openid",
        "offline_access",
        "Calendars.ReadWrite.Shared",
    ],
});
console.info("code", code);
```

One more thing worth mentioning maybe, is that we use Auth0 to do the authentication process.

**Comment by Nick-Lucas** [comment]
Sounds like this could be #2476 

Try dropping to version 2.3.0 for now

**Comment by tnorling** [comment]
@Nessario This was confirmed to be a bug in the library, the linked PR will address this in our next release. For now as @Nick-Lucas 
 mentioned, please downgrade to version 2.3.0

**Comment by Nessario** [comment]
Thanks @Nick-Lucas & @tnorling,

The `TypeError: errorCode is null` error was fixed by upgrading the library to the version 2.4.0, though it still didn't really help us much, because what we were looking for was the refresh token to send it to the back-end. Unfortunately, it looks like the *msal-browser* doesn't provide that field.

What we did instead is the [request an authorization code flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code) directly. In which we retrieve the `code` field from the redirect as response type of `fragment` and then attach it to our back-end request.
